src/app/section/FeatureSection/Features.module.css

```diff
@@ -1,3 +1,54 @@
+/* info Section Styles */
+
+.infoSection {
+  background-color: #f7f7f7;
+  color: #808080;
+  padding-top: 70px;
+  padding-bottom: 70px;
+}
+
+.infoContainer {
+  margin: 0 auto;
+}
+
+@media screen and (min-width: 200px) {
+  .infoContainer {
+    max-width: 90%;
+  }
+}
+
+@media screen and (min-width: 767px) {
```

Comment:
@Dheeru12503 
use media query for only 
1300px and 767px

Reply:
this kind of media query is not perfect for website and check reference website to understand media query.
In Reference website , 200px , 990 px of media query is also present. so . check it properly !!

Reply:
@Dheeru12503 
ok keep 900px but remove 200px  it's no use

Reply:
Done